### ISSUE TITLE
PropsSI API in Python 3.5 can't calculate mixture! 

### ISSUE THREAD

**Issue by zhenhaochu** [CoolProp/CoolProp#1633]
import CoolProp.CoolProp as CP
wf = "Nitrogen[0.75812]&water[0.04771]&oxygen[0.1375]&CO2[0.04703]&Argon[0.00964]"
CP.PropsSI("H", "P", 101325, "T", 373.15, wf)

==========
output:
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-2-99f05f0be08e> in <module>()
----> 1 CP.PropsSI("H", "P", 101325, "T", 373.15, "Nitrogen[0.75812]&water[0.04771]&oxygen[0.1375]&CO2[0.04703]&Argon[0.00964]")

CoolProp\CoolProp.pyx in CoolProp.CoolProp.PropsSI (CoolProp\CoolProp.cpp:41701)()

CoolProp\CoolProp.pyx in CoolProp.CoolProp.PropsSI (CoolProp\CoolProp.cpp:41489)()

CoolProp\CoolProp.pyx in CoolProp.CoolProp.__Props_err2 (CoolProp\CoolProp.cpp:39703)()

ValueError: One stationary point (not good) for T=373.15,p=101325,z=[ 0.83983835621, 4.27682163261e-05, 0.141212392808, 0.00955622480636, 0.00935025795987 ] : PropsSI("H","P",101325,"T",373.15,"Nitrogen[0.75812]&water[0.04771]&oxygen[0.1375]&CO2[0.04703]&Argon[0.00964]")

**Comment by ibell** [comment]
Unfortunately calculations with mixtures in CoolProp are not very reliable.  The algorithms are difficult to develop and make exceptionally robust.  The routines in NIST REFPROP are much more reliable.  Is this something you want to help us improve?

**Comment by zhenhaochu** [comment]
Recently, I replace my Python from version 2.7 to version 3.5, in Python 2.7 and C, the function PropsSI work well with mixture!

**Comment by ibell** [comment]
Does that same calculation work in C++?

**Comment by zhenhaochu** [comment]
Yes， It also work in Python2.7.

**Comment by ibell** [comment]
Can you please confirm that both 2.7 and 3.5 versions are using the same git revision?  You could show the output of:
``` python
import CoolProp
print(CoolProp.__version__)
print(CoolProp.__gitrevision__)
```

**Comment by zhenhaochu** [comment]
test result as below:
#python 3.5 output

import CoolProp

print(CoolProp.__version__)
6.1.0

print(CoolProp.__gitrevision__)
7864c2614ce5a0ef972386ee7f39859f0d3f8038

#######################

#python 2.7.11 output

in [1] import CoolProp

in [2] print CoolProp.__version__
out [2] 6.0.0

in [3] print CoolProp.__gitrevision__
out [3] df13df7756aa9352c56ef6f8d631365be590533c

in [4] import CoolProp.CoolProp as CP

in [5] CP.PropsSI("H", "P", 101325, "T", 373.15, "Nitrogen[0.75812]&water[0.04771]&oxygen[0.1375]&CO2[0.04703]&Argon[0.00964]")
out [5] 459222.3576869947

###############################
# change CoolProp to version 6.1.0 
# python 2.7.11 with CoolProp 6.1.0

in [1] import CoolProp

in [2] print CoolProp.__version__
out [2] 6.1.0

in [3] print CoolProp.__gitrevision__
out [3] 7864c2614ce5a0ef972386ee7f39859f0d3f8038

in [4] import CoolProp.CoolProp as CP

in [5] CP.PropsSI("H", "P", 101325, "T", 373.15, "Nitrogen[0.75812]&water[0.04771]&oxygen[0.1375]&CO2[0.04703]&Argon[0.00964]")

out [5] ValueErrorTraceback (most recent call last)
<ipython-input-5-99f05f0be08e> in <module>()
----> 1 CP.PropsSI("H", "P", 101325, "T", 373.15, "Nitrogen[0.75812]&water[0.04771]&oxygen[0.1375]&CO2[0.04703]&Argon[0.00964]")

CoolProp\CoolProp.pyx in CoolProp.CoolProp.PropsSI (CoolProp\CoolProp.cpp:41707)()

CoolProp\CoolProp.pyx in CoolProp.CoolProp.PropsSI (CoolProp\CoolProp.cpp:41495)()

CoolProp\CoolProp.pyx in CoolProp.CoolProp.__Props_err2 (CoolProp\CoolProp.cpp:39709)()

ValueError: One stationary point (not good) for T=373.15,p=101325,z=[ 0.83983835621, 4.27682163261e-005, 0.141212392808, 0.00955622480636, 0.00935025795987 ] : PropsSI("H","P",101325,"T",373.15,"Nitrogen[0.75812]&water[0.04771]&oxygen[0.1375]&CO2[0.04703]&Argon[0.00964]")

**Comment by zhenhaochu** [comment]
It seems Old version CoolProp can work with mixture properly! Is there something wrong with the C++ code update?

**Comment by ibell** [comment]
Yes, I fixed some problems, but created some new ones.  Doing flash
calculations with mixtures is very challenging !

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
